netbox_librenms_plugin/tests/test_coverage_api2.py

```
@@ -329,8 +332,10 @@ class _DoesNotExist(Exception):
         mock_job_cls.DoesNotExist = _DoesNotExist
         mock_job_cls.objects.get.return_value = mock_db_job
 
+        from rq.exceptions import NoSuchJobError
```

Flake8 (7.3.0)

[error] redefinition of unused 'NoSuchJobError' from line 321

(F811)